message_ix_community_meeting.rst

```diff
@@ -0,0 +1,82 @@
+.. _messageix workshop:
+
+MESSAGEix Community Meeting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Community Meeting aims to bring together researchers working on integrated assessment and/or energy system modeling using open source software. 
+
+It is a platform for showcasing new developments and features of the MESSAGEix modeling framework, sharing knowledge on various modeling approaches and applications of the framework, and discussing best practices in model-based policy analysis. The meeting is a place for facilitating collaboration between MESSAGEix users from around the world, by creating a forum where users can share their latest research outcomes and modeling experiences.
+
+The Community Meeting also aims to inspire new users of the MESSAGEix framework on the range of available possibilities and link them with other users in order to help them advance their modeling work.  
+For information about the model, its structure, mathematical
+formulation and much more, please see the `documentation <https://docs.messageix.org>`_.
```

Comment:
We can replace "model" with "modeling framework" and also add "installation" here and remove the section for installation on this page. So, this sentence will be: "For information about the modeling framework, its structure, mathematical formulation, installation, and much more, please see ..."